src/runtime_src/core/common/bo_cache.h

```diff
@@ -20,6 +20,7 @@
 #include <vector>
 #include <utility>
 #include <mutex>
+#include <sys/mman.h>
```

Comment:
sys/mman.h is not portable across windows. Have the clients include this file.

Reply:
I see this needed for Linux since the header files does call munmap.  On windows it doesn't though, because xrt.h on windows includes an munmap function defined in a header file included conditionally.   Leave this #include but guard it by #ifndef _WIN32

Reply:
> 
> 
> I see this needed for Linux since the header files does call munmap. On windows the include isn't needed here, because xrt.h on windows includes an munmap function defined in a header file included conditionally. Leave this #include but guard it by #ifndef _WIN32

Will fix with a new PR.